src/databricks/labs/ucx/source_code/languages.py

```diff
@@ -37,10 +37,10 @@ def fixer(self, language: Language, diagnostic_code: str) -> Fixer | None:
                 return fixer
         return None
 
-    def apply_fixes(self, language: Language, code: str) -> str:
+    def apply_fixes(self, language: Language, code: str, schema: str) -> tuple[str, str]:
```

Comment:
we need to introduce a new `CurrentSessionState` and inject it into two classes: `FromTable`, `SparkSql`. Do it from `Languages.__init__`. NOTE: _the instance of languages should be modified per notebook._

```
@dataclass
class CurrentSessionState:
   use_schema: str | None
```